### PR TITLE
950 hide sensitive data view capability page

### LINKED ISSUE #950: [FEATURE] Add hide icon for sensitive data in view Capability page.
Hide sensitive data like environment key-value pairs in capability view page. Have a eye-icon toggle functionality.

### PR DESCRIPTION
**Pull request by minump**
## Description
Hide sensitive data (environment variable value) in capability view page. Uses bootstrap eye icon for toggle.

**Fixes #950**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
@minump, please replace the env variable values with `*` when it's hidden (like how it appears when you add an env variable). Also, please add a space between the end of the env variable value and the eye icon so that the variable is more readable when displayed. Thanks.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added error message when the Contributions are not retrieved from the Contributions BB [#923](https://github.com/rokwire/rokwire-building-blocks-api/issues/923)
 - Delete contribution button. [#944](https://github.com/rokwire/rokwire-building-blocks-api/issues/944)
 - Add support for hybrid events in Events BB. [#955](https://github.com/rokwire/rokwire-building-blocks-api/issues/955)
+- Hide sensitive data in capability view page. [#950](https://github.com/rokwire/rokwire-building-blocks-api/issues/950)
```
Comment (sandeep-ps) [inline CHANGELOG.md:17]: Please move this to the Unreleased section. Thanks.

**Review by sandeep-ps — APPROVED** [review]
I tested the issue with steps to reproduce in the bug report, and the issue is fixed. I will approve this PR. Please fix the CHANGELOG. Thanks.

**Comment by minump** [comment]
> @minump, please replace the env variable values with `*` when it's hidden (like how it appears when you add an env variable). Also, please add a space between the end of the env variable value and the eye icon so that the variable is more readable when displayed. Thanks.

updated

File: CHANGELOG.md (hunk 1 above)
Comment (minump) [reply CHANGELOG.md]: updated

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
I'm merging this. Thanks.

**Pull request merged**